hudi-utilities/pom.xml

```diff
@@ -39,9 +39,10 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
         <configuration>
-          <source>1.8</source>
-          <target>1.8</target>
+          <source>${java.version}</source>
+          <target>${java.version}</target>
         </configuration>
```

Comment:
Does this even need to be redefined here ? Its already defined in root pom.

Reply:
responded in a above comment